build/release/Makefile

```diff
@@ -57,7 +57,7 @@ endif
 DOCKER_REGISTRY ?= rook
 QUAY_REGISTRY ?= quay.io/rook
 REGISTRIES ?= $(DOCKER_REGISTRY) $(QUAY_REGISTRY)
-IMAGES ?= rook toolbox
+IMAGES ?= rook toolbox flexvolume
```

Comment:
i thought about this but the flexvolume image is just a vehicle to carry the flex driver to the host. Because of this we dont need a base image with ceph stuff installed. Alpine would suffice.